x-pack/legacy/plugins/apm/server/routes/settings.ts

```diff
@@ -160,40 +160,45 @@ export function initSettingsApi(core: InternalCoreSetup) {
   });
 
   // Lookup single configuration
-  server.route({
-    method: 'POST',
-    path: `/api/apm/settings/cm/search`,
-    options: {
-      validate: {
-        query: {
-          _debug: Joi.bool()
-        }
+  [
+    '/api/apm/settings/agent-configuration/search',
+    '/api/apm/settings/cm/search' // backward compatible api route for apm-server
```

Comment:
this is the only old route i kept due to the dependency from apm-server. we can remove this route once they update their api call.

Reply:
We should apm-server folks to update it before the release so we don't have to support both endpoints for the life time of 7.x. cc @jalvz @simitt

Reply:
But good idea with the dual-routes for now 👍